content/administration/maintain/azure_oauth.rst

```diff
@@ -0,0 +1,212 @@
+called :guilabel:`Microsoft Outlook`.
+
+Next, navigate to :guilabel:`Settings --> General Settings`, and under the :guilabel:`Discuss`
+section, ensure that the checkbox for :guilabel:`Custom Email Servers` is checked. This populates
+a new option for :guilabel:`Outlook Credentials`.
+
+:guilabel:`Save` the progress.
+
+Then, copy and paste the :guilabel:`Client ID` (Application ID) and :guilabel:`Client Secret
+(Client Secret Value)` into the respective fields and :guilabel:`Save` the settings.
+
+.. image:: azure_oauth/outlookcreds.png
+   :align: center
+   :alt: Outlook Credentials in Odoo General Settings.
+
+Configure outgoing email server
+-------------------------------
+
+On the :guilabel:`General Settings` page, under the :guilabel:`Custom Email Servers` setting,
+click the :guilabel:`Outgoing Email Servers` link to configure the Microsoft account.
+
+Then, create a new email server and check the box for :guilabel:`Outlook`. Next, fill in the
+:guilabel:`Name` (it can be anything) and the Microsoft Outlook email :guilabel:`Username`.
+
+If the :guilabel:`From Filter` field is empty, enter either a :ref:`domain or email address
+<email_communication/default_from>`.
+
+Then, cick on :guilabel:`Connect your Outlook account`.
+
+A new window from Microsoft opens to complete the :guilabel:`authorization process`. Select the
+appropriate email address that is being configured in Odoo.
+
+.. image:: azure_oauth/verify-outlook.png
+   :align: center
+   :alt: Permission page to grant access between newly created app and Odoo.
+
+Then, allow Odoo to access the Microsoft account by clicking on :guilabel:`Yes`. After this, the
+page will navigate back to the newly configured :guilabel:`Outgoing Mail Server` in Odoo. The
+configuration automatically loads the :guilabel:`token` in Odoo, and a tag stating
+:guilabel:`Outlook Token Valid` appears in green.
+
+.. image:: azure_oauth/outlook-token.png
+   :align: center
+   :alt: Valid Outlook Token indicator.
+
+Finally, click :guilabel:`Test Connection`. A confirmation message should appear. The Odoo database
+can now send safe, secure emails through Microsoft Outlook using OAuth authentication.
+
+Multiple user configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Each user should have a separate server setup and the :guilabel:`from-filter` should be set so that
+only the user's email will be sent from that server. In other words, only a user with a matching
+email address will be able to use this server.
+
+Then, set up a fall-back account email to allow for the sending of :guilabel:`notifications`. This
+email would be configured as a :guilabel:`general transactional server`. The
+:guilabel:`mail.default.from` system parameter would need to be set to the :guilabel:`username` of
+the general transactional server account. For more information see :ref:`Use a default email
+address <email_communication/email_domain/default>`.
```

Comment:
```suggestion
Each user should have a separate server set up. The :guilabel:`from-filter` should be set so that
only the user's email will be sent from that server. In other words, only a user with an email 
address that matches the set :guilabel:`from-filter` will be able to use this server.

After setting the :guilabel:`from-filter`, set up a fallback email account to allow for the sending
of :guilabel:`notifications`. The fallback email must be configured as a :guilabel:`general
transactional server`. The :guilabel:`mail.default.from` system parameter must be set to the
:guilabel:`username` of the general transactional server account. For more information see :ref:`Use
a default email address <email_communication/email_domain/default>`.
```